Tests/run_tests.py

```diff
@@ -68,9 +68,8 @@ def is_pypy():
 # Following modules have historic failures. If you fix one of these
 # please remove here!
 EXCLUDE_DOCTEST_MODULES = [
-    "Bio.PDB",
-    "Bio.PDB.AbstractPropertyMap",
-]
+    ]
```

Comment:
You can put the closing ``]`` on the line above (also as it is, the indentation is strange).